server/registry/internal/storage/gorm/client.go

```diff
@@ -116,9 +116,8 @@ func (c *Client) ensureTable(v interface{}) error {
 	return nil
 }
 
-// EnsureTables ensures that all necessary tables exist in the database.
-func (c *Client) EnsureTables() error {
-	entities := []interface{}{
+func (c *Client) entities() []interface{} {
```

Comment:
Can this be moved into a package level variable now?

Reply:
Sure, I'll do that.

Reply:
done